Azure/azure-cli
# [Storage] `az storage account show-connection-string`: Append endpoints by default

**Related command**

`az storage account show-connection-string`

**Description**

Automatically append endpoints in the returned connection string to fix https://msazure.visualstudio.com/One/_workitems/edit/14121590

**Testing Guide**

`az storage account show-connection-string -n yssa -g yishitest`

Output before:
```
{
  "connectionString": "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=yssa;AccountKey=Redacted==;"
}
```
Output after:
```
{
  "connectionString": "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=yssa;AccountKey=Redacted==;BlobEndpoint=https://yssa.blob.core.windows.net/;FileEndpoint=https://yssa.file.core.windows.net/;QueueEndpoint=https://yssa.queue.core.windows.net/;TableEndpoint=https://yssa.table.core.windows.net/"
}
```


---

This checklist is used to make sure that common guidelines for a pull request are followed.
